apps/st2-rules/controller.js

```diff
@@ -20,10 +20,10 @@ angular.module('main')
         url: '/new'
       })
       .state('rules.general', {
-        url: '/{ref:\\w+}/general?edit'
+        url: '/{ref:[\\w.-]+}/general?edit'
```

Comment:
Don't the dot and minus need to be escaped since we want to use it as a literal character?